client/login/wp-login/components/one-login-layout.scss

```diff
@@ -12,6 +12,21 @@
 	text-align: center;
 }
 
+/**
+ * By default, all headings get .wp-brand-font class and it applies a 40px font size. That size should only be used
+ * for Recoleta font. Any other heading, should be 32px. This rule achieves this affect.
+
+ * TODO: to clean up this code, we should apply .wp-brand-font selectively, only when we use Recoleta.
```

Comment:
I wasn't sure to do this TODO now or later. It feels bad to add an exception in one-login-layout.scss, which was supposed to be unified